qiskit_dynamics/backend/backend_utils.py

```diff
@@ -167,3 +168,92 @@ def _sample_probability_dict(
 def _get_counts_from_samples(samples: list) -> Dict:
     """Count items in list."""
     return dict(zip(*np.unique(samples, return_counts=True)))
+
+
+def _get_subsystem_probabilities(probability_tensor: np.ndarray, sub_idx: int) -> np.ndarray:
+    """Marginalize a probability vector according to subsystems.
+
+    Args:
+        probability_tensor: Probability full system reshaped to reversed(subsystem_dims).
```

Comment:
This doc string is unclear:

```suggestion
        probability_tensor: Probability full system reshaped to reversed(subsystem_dims).
```

This method doesn't know about `subsystem_dims`. Maybe say somethign like: 

```
k-dimensional probability array, where the probability of outcome `(idx1, ..., idxk)` is `probability_tensor[idx1, ..., idxk]`. 
```
I guess you also need to be clear that the `sub_idx` itself is in reverse order (which is what your original comment was alluding to).

Reply:
done